README.md

```diff
@@ -596,8 +624,12 @@ With example of: `train.txt`, `obj.names`, `obj.data`, `yolo-obj.cfg`, `air`1-6`
 
 ## How to use Yolo as DLL and SO libraries
 
-* on Linux - set `LIBSO=1` in the `Makefile` and do `make`
-* on Windows - compile `build\darknet\yolo_cpp_dll.sln` or `build\darknet\yolo_cpp_dll_no_gpu.sln` solution
+* on Linux
+    * build `darknet` using `cmake` or
+    * set `LIBSO=1` in the `Makefile` and do `make`
+* on Windows
+    * compile `build\darknet\yolo_cpp_dll.sln` solution or
```

Comment:
also on windows you can build darknet using cmake (and obtain dll). I'd put the two solutions (yolo_cpp_dll.sln and yolo_cpp_dll_no_gpu.sln) under the same bullet, so that we have a symmetry between linux and windows: both have cmake, one has make, one has VS

Reply:
I see so something like this:
```
* on Windows
    * build `darknet` using `Cmake-GUI`
    * compile `build\darknet\yolo_cpp_dll.sln` or `build\darknet\yolo_cpp_dll_no_gpu.sln` solution
```

Reply:
I'd not rely too much on CMake gui... for a newcomer it might be "too much" complicated. Also, CMake gui exists for linux too! and works there also!! so if you want to discuss it, please always consider cmake and anything related to it as the only method that works in the same way with the same results on any system and any compiler (gcc, clang, msvc, ...)

I don't know, but for me the build scripts (build.ps1 for windows and build.sh for linux/mac) should be preferred by novice users...

Reply:
Yeah, I agree with that.
So I'll just use `cmake` instead of `cmake-gui`
```
* on Windows
    * build `darknet` using `cmake` or
    * compile `build\darknet\yolo_cpp_dll.sln` or `build\darknet\yolo_cpp_dll_no_gpu.sln` solution
```
For the build scripts should we mention them as well then?
```
* on Windows
    * using `build.ps1` or
    * build `darknet` using `cmake` or
    * compile `build\darknet\yolo_cpp_dll.sln` or `build\darknet\yolo_cpp_dll_no_gpu.sln` solution
```

Reply:
@cenit I went ahead and added the commit.